content/open-source/sigstore/cosign/how-to-verify-file-signatures-with-cosign.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "How to Verify File Signatures with Cosign"
+type: "article"
+description: "Use Cosign to verify non-container software artifacts"
+lead: "Cosign can verify software artifacts beyond containers"
+date: 2022-12-21T15:22:20+01:00
+lastmod: 2022-12-21T15:22:20+01:00
+draft: false
+images: []
+menu:
+  docs:
+    parent: "cosign"
+weight: 005
+toc: true
+---
+
+Cosign can verify more than just containers. Blobs, or binary large objects, and standard files can be verified in a similar way. You can publish a blob or other artifact to an OCI (Open Container Initiative) or elsewhere and then use Cosign to verify its signature. This tutorial assumes you have Cosign installed, which you can achieve by following our [How to Install Cosign guide](../how-to-install-cosign/).
+
+We’ll verify a binary artifact, in this case, a copy of [`apko`](/open-source/apko/overview/), which is a command-line tool for building container images using a declarative language based on YAML. The methods in this tutorial apply to any blob file Cosign has signed with a keyless signature.
```

Comment:
I copied most of this intro from https://edu.chainguard.dev/open-source/sigstore/cosign/how-to-sign-blobs-with-cosign/

Reply:
I took a tab at rewriting it; we want to have different language so that it's not confusing with the other article, so that it does not negatively impact SEO, and so that the reader can understand what is at stake here. I think it probably could be stronger on this last point